linera-core/src/local_node.rs

```diff
@@ -250,7 +250,7 @@ where
             .await?
             .manager
             .get()
-            .pending_blobs
```

Comment:
If I'm reading this correctly, `LocalNode::find_missing_blobs` is used in `send_certificate`, which in turn is used with different certificate values:

 sometimes sends `ConfirmedBlock`s, `Timeout`s and `ValidatedBlock`s (which are, then, always the locked block in the local node, because they are in the current round, I believe?).

* If it sends a `ConfirmedBlock`, the blobs must already be in local storage.
* If it sends a `Timeout`, there can't be any blobs.
* If it sends a `ValidatedBlock`, this will (I think?) always be the local node's locked block, so in that case the blobs will _always_ be exactly those in the chain manager.

Maybe we should look up the blobs only in the expected places in each case?